<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="yogesh2693" failures="3" tests="3" name="com.SauceDemo.TestClasses.TC02Singleproduct" time="10.375" errors="0" timestamp="2022-05-28T12:35:02 IST" skipped="0">
  <testcase classname="com.SauceDemo.TestClasses.TC02Singleproduct" name="singleproducttest" time="2.693">
    <failure message="test fail if expected and actual not equal expected:&lt;[0]&gt; but was:&lt;[1]&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: test fail if expected and actual not equal expected:<[0]> but was:<[1]>
at junit.framework.Assert.assertEquals(Assert.java:100)
at com.SauceDemo.TestClasses.TC02Singleproduct.singleproducttest(TC02Singleproduct.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- singleproducttest -->
  <system-out/>
  <testcase classname="com.SauceDemo.TestClasses.TC02Singleproduct" name="singleproducttest" time="4.217">
    <failure message="test fail if expected and actual not equal expected:&lt;[0]&gt; but was:&lt;[1]&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: test fail if expected and actual not equal expected:<[0]> but was:<[1]>
at junit.framework.Assert.assertEquals(Assert.java:100)
at com.SauceDemo.TestClasses.TC02Singleproduct.singleproducttest(TC02Singleproduct.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- singleproducttest -->
  <system-out/>
  <testcase classname="com.SauceDemo.TestClasses.TC02Singleproduct" name="singleproducttest" time="3.465">
    <failure message="test fail if expected and actual not equal expected:&lt;[0]&gt; but was:&lt;[1]&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: test fail if expected and actual not equal expected:<[0]> but was:<[1]>
at junit.framework.Assert.assertEquals(Assert.java:100)
at com.SauceDemo.TestClasses.TC02Singleproduct.singleproducttest(TC02Singleproduct.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- singleproducttest -->
  <system-out/>
</testsuite> <!-- com.SauceDemo.TestClasses.TC02Singleproduct -->
